cfgov/filing_instruction_guide/jinja2/filing_instruction_guide/index.html

```diff
@@ -81,7 +81,7 @@ <h2 class="appendix-header">Additional Information</h2>
         !function(){
           {# Include site-wide JavaScript. #}
           var s = [
-            '{{ static('apps/research-reports/js/index.js') }}',
+            '{{ static('apps/filing-instruction-guide/js/fig-sidenav.js') }}',
```

Comment:
This is referencing `apps/filing-instruction-guide/js/fig-sidenav.js`, but the file appears to have been moved to `apps/filing-instruction-guide/fig-sidenav.js` (without the `/js/`). Looks like the moved file should be moved into a `js` directory in its new location.

Reply:
I'm seeing the JS file in the right place. I think it's at `apps/filing-instruction-guide/js/fig-sidenav.js` now, which is right, right?

Reply:
PS, thank you @anselmbradford for reviewing!